indico/web/templates/forms/checkbox_group_widget.html

```diff
@@ -6,9 +6,15 @@
     {% set field_disabled = input_args.pop('disabled', false) %}
     {% for subfield in field %}
         {% set disabled = field_disabled or subfield.data in field.disabled_choices|default([]) %}
-        <div class="i-checkbox {{ 'checkbox-disabled' if disabled}}">
-            {{ subfield(disabled=disabled, **input_args) }}
-            {{ subfield.label }}
+        {% set reason = field.disabled_choices_reasons.get(subfield.data) %}
```

Comment:
This field is used elsewhere (e.g. when managing oauth apps) where `disabled_choices_reasons` is not defined

Reply:
Thanks, fixed :)